OpenRA.Mods.Common/Scripting/ScriptTriggers.cs

```diff
@@ -421,6 +422,48 @@ public void OnDiscovered(Actor self, Player discoverer, bool playNotification)
 			}
 		}
 
```

Comment:
Here too please, and then add the attribute that requires an explicit implementation. Sorry, I assumed that was obvious from the original discussion, but I guess not.